components/Layout.js

```diff
@@ -0,0 +1,14 @@
+import Footer from "./Footer";
+import Navbar from "./Navbar";
+
+const Layout = ({ children }) => {
+  return (
+    <div className="content">
+        <Navbar />
+        { children }
+        <Footer />
+    </div>
+  );
+}
+
+export default Layout;
```

Comment:
Nice implementation of a Layout component inside _app for custom layout on every page. Components with clean functional code.